cmd/anonymous-main.go

```diff
@@ -79,8 +79,8 @@ EXAMPLES:
   4. Set anonymous to "public" for bucket with prefix on Amazon S3 cloud storage.
      {{.Prompt}} {{.HelpName}} set public s3/public-commons/images
 
-  5. Set a custom prefix based bucket anonymous on Amazon S3 cloud storage using a JSON file.
-     {{.Prompt}} {{.HelpName}} set-json s3/public-commons/images /path/to/anonymous.json
+  5. Set bucket to "custom" on Amazon S3 cloud storage using a JSON file.
```

Comment:
I think old description looks fine. Just uses command should change.

Reply:
I reverted to the old description and just swapped the command, also in the comment on the bottom of the file